openshift/release/release-notes.md

```diff
@@ -0,0 +1,299 @@
+
+* Allow secure connection to eventlistener pod ([#819](https://github.com/tektoncd/triggers/pull/819))
+
+  HTTPS connection to eventlistener can be configured by tweaking eventlistener configuration
+  ```yaml
+   apiVersion: triggers.tekton.dev/v1alpha1
+   kind: EventListener
+   metadata:
+     name: github-listener-interceptor
+   spec:
+     ...
+     resources:
+       kubernetesResource:
+         spec:
+           template:
+             spec:
+               serviceAccountName: tekton-triggers-github-sa
+               containers:
+               - env:
+                 - name: TLS_CERT
+                   valueFrom:
+                     secretKeyRef:
+                       name: tls-key-secret
+                       key: tls.crt
+                 - name: TLS_KEY
+                   valueFrom:
+                     secretKeyRef:
+                       name: tls-key-secret
+                       key: tls.key
+  ```
+* Drop escaping of strings in the JSON ([#823](https://github.com/tektoncd/triggers/pull/823))
+  Change the escaping of parameters into TriggerTemplates.
+
+## Backwards incompatible changes :rotating_light:
+In the current release:
+
+* TEP-0022: Switch to immutable input event bodies ([#828](https://github.com/tektoncd/triggers/pull/828))
+
+  action required: If you are using overlays in the CEL Interceptor, please update your bindings to use $(extensions.) instead of $(body.)
+
+  BREAKING CHANGE:
+  
+  CEL overlays now add fields to a new top level extensions field instead of the modifying the incoming event body. TriggerBindings can access values within this new extensions field using `$(extensions.<key>)` syntax.
+
+* Drop escaping of strings in the JSON ([#823](https://github.com/tektoncd/triggers/pull/823))
+
+  Previously, parameters were escaped as they were being replaced into a TriggerTemplate, by replacing double-quotes " with an escaped version ", this functionality has been removed, as it was breaking quoted strings, and in some cases, rendering the resulting output unparseable.
+
+  action required: If you were relying on the escaping, you can retain the old behaviour by adding an annotation to an affected TriggerTemplate, `triggers.tekton.dev/old-escape-quotes: "true"`
+
+## Fixes :bug:
+* Don't log Fatal when handling events in the eventlistener ([#810](https://github.com/tektoncd/triggers/pull/810))
+* Fix apiVersion for test triggerbinding ([#826](https://github.com/tektoncd/triggers/pull/826))
+* Change the webhook name to triggers-webhook ([#829](https://github.com/tektoncd/triggers/pull/829))
+ 
+  Fix an issue that caused the webhook, under certain conditions, to fail to acquire a lease and not function correctly as a result.
+* Fix EL Container Arguments in Reconciler ([#838](https://github.com/tektoncd/triggers/pull/838))
```

Comment:
How about removing entire `Fixes`?

Reply:
@savitaashture before doing that, is there any of these fix fixing a bug that was reported / mentionned by customers (jira, …) ?

Reply:
no 
There are no such bugs